prototype/core/impl/src/main/java/org/eclipse/sensinact/prototype/model/nexus/impl/NexusImpl.java

```diff
@@ -64,9 +76,83 @@ public NexusImpl(ResourceSet resourceSet, SensiNactPackage sensinactPackage,
         this.resourceSet = resourceSet;
         this.sensinactPackage = sensinactPackage;
         this.notificationAccumulator = accumulator;
+        // TODO we need a general Working Directory for such data
+        Optional<EPackage> packageOptional = loadDefaultPackage(BASIC_BASE_ECORE);
 
-        defaultPackage = EMFUtil.createPackage("base", DEFAULT_URI, "sensinactBase", this.resourceSet);
+        defaultPackage = packageOptional
+                .orElseGet(() -> EMFUtil.createPackage("base", DEFAULT_URI, "sensinactBase", this.resourceSet));
         packageCache.put(DEFAULT_URI_OBJECT, defaultPackage);
+        loadInstances();
+    }
+
+    private Optional<EPackage> loadDefaultPackage(String fileName) {
+        Resource resource = resourceSet.createResource(URI.createFileURI(fileName));
+        if (new File(fileName).exists()) {
+            try {
+                resource.load(null);
+                if (!resource.getContents().isEmpty()) {
+                    EPackage defaultPackage = (EPackage) resource.getContents().get(0);
+                    resource.setURI(URI.createURI(defaultPackage.getNsURI()));
+                    return Optional.of(defaultPackage);
+                }
+            } catch (IOException e) {
+                Exceptions.duck(e);
+            }
+        }
+        return Optional.empty();
+    }
+
+    private void loadInstances() {
+
+        Path instancesPath = Paths.get(INSTANCES);
+        if (instancesPath.toFile().exists()) {
+            try {
+                Files.walk(instancesPath).forEach(this::load);
+            } catch (IOException e) {
+                Exceptions.duck(e);
+            }
```

Comment:
Don't duck the exception. Handle it and make a decision to fail the whole load, or log something